inst/extdata/gcam-china/CCS_provincial_data_add.csv

```diff
@@ -0,0 +1,19 @@
+# File: CCS_provincial_data_add.csv
+# Title: CO2 and Cost(1990USD) by province
+# Units: CO2-Mt,Cost-1990USD per t CO2
+# Description: cost grades of provinces without data
```

Comment:
# Description: cost grades of provinces without data (Tibet and Hainan) or onshore storage (Fujian, Guangdong, Guangxi, and Qinghai)

Reply:
get